airbyte-integrations/connectors/source-twilio/source_twilio/source.py

```diff
@@ -60,8 +60,13 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
                 config["auth_token"],
             ),
         )
-        full_refresh_stream_kwargs = {"authenticator": auth}
-        incremental_stream_kwargs = {"authenticator": auth, "start_date": config["start_date"]}
+        full_refresh_stream_kwargs = {"authenticator": auth, "pagesize": config["page_size"]}
```

Comment:
```suggestion
        full_refresh_stream_kwargs = {"authenticator": auth}
```
As discussed in another comment, I don't think page size should be set from the config.